RZTouchID/RZTouchID.h

```diff
@@ -28,6 +28,43 @@
 
 #import <Foundation/Foundation.h>
 
+
+/**
+ *  RZTouchIDDelegate allows you to provide a custom mechanism for storing the touch ID enabled userID.
+ * 
+ *  Typically you might store the username using NSUserDefaults but you have freedom to implement this in however you see fit.  
+ *  
+ *  Be aware that if the mechanism you use is replicated to the cloud the userId might persist onto other devices but the keychain might not. In this case the user would experience login failures and have to manually enter the password.
+ */
+@protocol RZTouchIDDelegate <NSObject>
+
+/**
+ *  Called by deletePasswordWithIdentifier:completion: 
+ * 
+ *  You should remove the provided UserID from your store of touch ID enabled User IDs.
+ *
+ *  @param touchIDUserID A string representing a user ID. This could be an email, a UUID or a username.
+ */
+- (void)disableTouchIDForUserID:(NSString *)touchIDUserID;
```

Comment:
This should be named `touchID:(RZTouchID *)touchID didDeletePasswordForIdentifier:(NSString *)identifier;`